app/models/request.rb

```diff
@@ -69,7 +71,7 @@ def messages_by_contributor
   def self.broadcast!(request)
     if request.planned?
       BroadcastRequestJob.delay(run_at: request.schedule_send_for).perform_later(request.id)
-      RequestScheduled.with(request_id: request.id).deliver_later(User.all)
+      RequestScheduled.with(request_id: request.id, organization_id: request.organization.id).deliver_later(User.all)
```

Comment:
Why have organization_id as its own parameter if we can deduct it from the request itself? Doesn't that add a possible source of errors?

Reply:
See :point_up: